data/nlu.md

```diff
@@ -291,6 +318,37 @@
 - i would like to try [chinese](cuisine) food
 - get me some [Sri Lankan](cuisine) [Sea food](cuisine) [Mexican](cuisine) [Desserts](cuisine) [North Indian](cuisine) [Awadhi](cuisine) [Bihari](cuisine) [Mishti](cuisine) [Turkish](cuisine) [Tibetan](cuisine) [Cantonese](cuisine) [Rajasthani](cuisine) [Salad  ](cuisine)[Paan](cuisine)
 - i would like to eat [Konkan](cuisine) [Russian](cuisine) [Bubble Tea](cuisine) [Nepalese](cuisine) [Bangladeshi](cuisine) [Singaporean](cuisine) [Fried Chicken](cuisine)[Fried](category) Chicken [Mughlai](cuisine) [Kerala](cuisine) [Greek](cuisine) [Steak](cuisine) [Lucknowi](cuisine) [Roast Chicken](cuisine) [Andhra](cuisine) [Lebanese](cuisine) food
+- i would prefer [chinese](cuisine) food
+- [italian](cuisine) please
+- [chinese](cuisine) food
+- get me some [chinese](cuisine) food
+- get me some [italian](cuisine) food please
+- i would love to eat some [italian](cuisine) food
+- [chinese](cuisine) food please
+- [italian](cuisine) food
+- i will prefer [north indian](cuisine) food
+- i would like to eat [north indian](cuisine) food
+- i would prefer to eat [chinese](cuisine) food
+- [north indian](cuisine) food
+
+## intent:telling_datetime
+- please book for tomorrow
+- please book for [tomorrow 8 pm](time:2019-10-02T20:00:00.000+00:00)
+- tomorrow please
+- tomorrow evening
+- day after tomorrow please
+- please book for 10 october 2019
+- book for 1 january 2020
+- i would like to book for next week
+- please book for next week
+- can you please book for today itself
+- please book it for [tomorrow](time:2019-10-02T00:00:00.000+00:00)
+- book it for [tomorrow](time:2019-10-02T00:00:00.000+00:00) please
+- book it for [tomorrow](time:2019-10-03T00:00:00.000+00:00)
+- book for [tomorrow](time:2019-10-03T00:00:00.000+00:00)[tomorrow](time:2019-10-04T00:00:00.000+00:00)
+- book it for [today](time:2019-10-03T00:00:00.000+00:00)
+- book it for [today](time:2019-10-03T00:00:00.000+00:00) only
+- book for [friday 8 pm](time:2019-10-04T20:00:00.000+00:00)
```

Comment:
The current format of the time is good? Or shall we change to something else? like one entity for day and one for time.